### PR TITLE
Issue 113 revisited

### PR DESCRIPTION
**Pull request by hsolbrig**
Pull request to check changes to biolink-model

### CODE REVIEW

File: biolinkml/utils/schemaloader.py [hunk 1]
```diff
@@ -195,7 +195,7 @@ def resolve(self) -> SchemaDefinition:
         for slot in self.schema.slots.values():
             # Propagate domain to containing class
             if slot.domain and slot.domain in self.schema.classes:
-                if slot.name not in self.schema.classes[slot.domain].slots:
+                if slot.name not in self.schema.classes[slot.domain].slots and not slot.owner:
                     slot.owner = slot.name
                     self.schema.classes[slot.domain].slots.append(slot.name)
```
Comment (cmungall) [inline biolinkml/utils/schemaloader.py:200]: I'm still suspicious of lines like this

Comment (cmungall) [reply biolinkml/utils/schemaloader.py:200]: this line: `self.schema.classes[slot.domain].slots.append(slot.name)`

Comment (hsolbrig) [reply biolinkml/utils/schemaloader.py:200]: Why?

Comment (cmungall) [reply biolinkml/utils/schemaloader.py:200]: it looks like domains are inducing classes to have slots?

**Pull request merged**